app/assets/stylesheets/login.css

```
@@ -0,0 +1,9 @@
+.login-form { width: 385px; margin: 20px auto; }
```

Sugiro deixar essas regras em múltiplas linhas, para ficar mais fácil de ler (vale para todas):
```suggestion
.login-form {
  width: 385px; 
  margin: 20px auto; 
}
```